gateway/src/apicast/policy/custom_metrics/Readme.md

```diff
@@ -0,0 +1,73 @@
+# Custom metrics policies
+
+This policy adds the availability to add metrics after the Upstream API
+response.  The main use case for this policy is to add metrics based on response
+code status, headers or different Nginx variables.
+
+## Caveats:
+
+Due to this needs some information from the upstream API, if the authrep with
```

Comment:
I think it would be better to use the authorize and report endpoints instead of authrep.
That way we could authorize => call upstream => report. The problem with the current approach is that it can lose many reports.